source/d3d12/d3d12.cpp

```diff
@@ -6,6 +6,9 @@
 #include "log.hpp"
 #include "hook_manager.hpp"
 #include "d3d12_device.hpp"
+#include "runtime_d3d12.hpp"
+
+#undef ID3D12GraphicsCommandList_ClearDepthStencilView
```

Comment:
The changes in this file are not used (probably leftovers from earlier).

Reply:
Yes, it can be removed =)

Reply:
Yes. It can be removed